docs/reference/esql/esql-kibana.asciidoc

```diff
@@ -261,7 +261,10 @@ of rows that are retrieved by the query and displayed in Discover. Queries and
 aggregations run on the full data set.
 * Discover shows no more than 50 columns. If a query returns
```

Comment:
by the way, I didn't hit this column limitation during my testing. It may not exist and there are no column limits.

Reply:
It is not a limitation of ES but of Discover. If you run keep * and you have more than 50 you will see 50 columns in the table selected and not all of them. \

For example:
<img width="2500" alt="image" src="https://github.com/elastic/elasticsearch/assets/17003240/de81ea93-dd24-4421-a57a-fdf5f51ced46">

Reply:
Aha! I didn't get it because I was adding them "manually" and it allowed past 50 

![Screenshot 2024-01-18 at 15 17 47](https://github.com/elastic/elasticsearch/assets/7784120/ac93c96d-9b78-4c08-b11b-0ad97d0aa35d)

Ok... got it

Reply:
Yes it is about the programmatically added by default, the user can do whatever they want!